app/api-v1/controllers/Attachment/index.js

```diff
@@ -1,5 +1,72 @@
+const { parseAccept } = require('../../../utils/parseAcceptHeader')
+const logger = require('../../../utils/Logger')
+
+const db = require('../../../db')
+
+const { createAttachmentFromFile, createAttachment } = require('../Attachment/helpers')
+const { InternalError, NotFoundError, BadRequestError } = require('../../../utils/errors')
+
 module.exports = {
-  getAll: async function () {
-    return { status: 500, response: { message: 'Not Implemented' } }
+  get: async (req) => {
+    const { id } = req.params
+    if (!id) throw new BadRequestError('missing params')
+
+    const [attachment] = await db.getAttachment(req.params.id)
+    if (!attachment) throw new NotFoundError('Attachment Not Found')
+    const orderedAccept = parseAccept(req.headers.accept)
+
+    for (const mimeType of orderedAccept) {
+      if (mimeType === 'application/octet-stream' && attachment.filename.includes('json'))
+        return {
```

Comment:
`return{}` be replaced by `throw new NotAcceptableError('Client file request not supported')`